ci/ray_ci/automation/pypi_lib.py

```diff
@@ -0,0 +1,60 @@
+import subprocess
+import os
+from typing import List
+
+from ray_release.aws import get_secret_token
+
+AWS_SECRET_TEST_PYPI = "ray_ci_test_pypi_token"
+AWS_SECRET_PYPI = "ray_ci_pypi_token"
+
+bazel_workspace_dir = os.environ.get("BUILD_WORKSPACE_DIRECTORY", "")
+
+
+def _get_pypi_url(pypi_env: str) -> str:
+    if pypi_env not in ["test", "prod"]:
+        raise ValueError(f"Invalid pypi_env: {pypi_env}")
```

Comment:
have a helper (like `_check_pypi_Env()`) to avoid repeating these two lines in the two functions?

Reply:
Done